### PR TITLE
fix: validate params

### PR DESCRIPTION
**Pull request by lfbdev**
## Description

auto-release-darwin-arm64.sh should check all variables. If empty, no error should occur.

### Related Issues

#427 

### Current Behavior

image

### CODE REVIEW

File: build/package/auto-release-darwin-arm64.sh [hunk 1]
```diff
@@ -10,6 +10,16 @@ plugin_dir=.devstream
 GOOS=$(go env GOOS)
 GOARCH=$(go env GOARCH)
 
+if [ ! $tag ] || [ ! $user ] || [ ! $repo ] || [ ! $github_token ] || [ ! $plugin_dir ]; then
+  echo "The following variables cannot be empty!"
+  echo "tag="$tag
+  echo "user="$user
+  echo "repo="$repo
+  echo "github_token="$github_token
```
Comment (IronCore864) [inline build/package/auto-release-darwin-arm64.sh:18]: if github_token isn't empty but maybe for example, tag is empty, github_token will be printed out on the screen, which isn't safe.

Comment (daniel-hutao) [reply build/package/auto-release-darwin-arm64.sh:18]: +1

**Review by daniel-hutao — CHANGES_REQUESTED** [review]
set -o errexit
set -o nounset
set -o pipefail

File: build/package/auto-release-darwin-arm64.sh (hunk 1 above)
Comment (lfbdev) [reply build/package/auto-release-darwin-arm64.sh]: 1.  add "set -o nounset" for undefined parameter
2. do not  print the true token

**Review by daniel-hutao — APPROVED** [review]

**Pull request merged**